RTree.h

```diff
@@ -681,6 +690,52 @@ bool RTREE_QUAL::LoadRec(Node* a_node, RTFileStream& a_stream)
 }
 
 
+RTREE_TEMPLATE
+void RTREE_QUAL::CopyRec(Node* current, Node* other)
+{
+  current->m_level = other->m_level;
+  current->m_count = other->m_count;
+
+  if(current->IsInternalNode())  // not a leaf node
```

Comment:
I think that you could remove some code duplication here by putting the 'if... is internal node' statement inside the for loop instead of the other way around.

Reply:
Absolutely correct, I just copied `LoadRec` and did not refactor. Should I fork again and fix it? Thanks for merging, by the way 👍